Roadmap.md

```diff
@@ -25,7 +27,7 @@
 - **No direct PR's into master branch.** It will be locked and only merges from stable development branches will be taken.
 - Exception will be major bug fixes on a case by case basis.
 - Please use the [working development branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch) for daily feature work.
-- [Working development branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch) will be snapped every 3 weeks for stabilization.
+- [Working development branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch) will be snapshot every 3 weeks for stabilization into a separate branch. Working branch will still be open for contributions.
 - That stabilization branch will be worked on for 1 week.
```

Comment:
"at least 1 week" might be better. some releases could take more time to stabilize.

Reply:
Great suggestion. Will update.